package.json

```diff
@@ -29,7 +29,7 @@
     "babel-eslint": "^10.0.1",
     "babel-loader": "^8.0.0",
     "chart.js": "^2.7.3",
-    "copy-webpack-plugin": "^4.6.0",
+    "copy-webpack-plugin": "^5.0.5",
```

Comment:
This is used to copy `.md` and `.php` files into the build output. Even though this is a major version bump, none of the breaking changes appear to apply:

https://github.com/webpack-contrib/copy-webpack-plugin/releases/tag/v5.0.0

All of the expected files are in the build output, as well.